althea/althea-rust-binaries-test/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2017-2018 Justin Kilpatrick, Stanislaw Drozd, Openwrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=althea-rust-binaries-test
+PKG_VERSION:=0.1
+PKG_RELEASE:=0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=http://github.com/althea-mesh/althea_rs
+PKG_SOURCE_VERSION:=master
+PKG_LICENSE:=Apache-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/althea-rust-binaries
+  SECTION:=net
+  CATEGORY:=Utilities
+  SUBMENU:=Althea
+  TITLE:=Althea Rust executables
+  URL:=https://github.com/althea-mesh/althea_rs
+  MAINTAINER:= Justin Kilpatrick <justin@altheamesh.com>
+  DEPENDS:=@IPV6 +libopenssl +libsqlite3
+endef
+
+define Package/althea-rust-binaries/description
+  Althea Rust executables
+endef
+
+ifeq ($(ARCH),arm)
+	RUST_TRIPLE:=arm-unknown-linux-musleabihf
+else ifeq ($(ARCH),mips)
+	RUST_TRIPLE:=mips-unknown-linux-musl
+else ifeq ($(ARCH),mipsel)
+	RUST_TRIPLE:=mipsel-unknown-linux-musl
```

Comment:
are we going to support x86 openwrt or just x86 on desktop linux?

Reply:
we can support x86 openwrt I can make changes for that.